radicle-crdt/src/redactable.rs

```diff
@@ -0,0 +1,80 @@
+use crate::Semilattice;
+
+/// An object that can be either present or removed.
+///
+/// The "redacted" state is the top-most element and takes precedence
+/// over other states.
+///
+/// There is no `Default` instance, as it wouldn't obbey semilattice laws.
```

Comment:
Just double-checked, the semilattice doesn't need an identity element. If it does have one then it's a bounded semilattice.

Reply:
Aha, then I'll rephrase that to "This semilattice is not bounded" or something.